facia/app/controllers/FaciaController.scala

```diff
@@ -155,6 +155,11 @@ trait FaciaController
       Cached(CacheTime.Facia)(WithoutRevalidationResult(Found(LinkTo(s"/$path$params"))))
     }
 
+  // Returns a 'lite' version of the 'lite' version of a PressedPage.
```

Comment:
yeah.. 🤔 I know it's not the clearest phrasing but I think it captures how the terms are used in the code as things stand. (Both `renderFrontJson` and `renderFrontJsonLite` request the same `lite` version of the `PressedPage` from S3.)

Reply:
Oh hah, I thought it was a typo!

Maybe a `Returns a reduced version of the 'lite' version of a PressedPage.` 🤔 not sure if thats any better though!

Reply:
Ah, got it! I've added another commit to rename this method using 'minimal' instead of 'lite'. What do you think? It's still not very descriptive, but it at least avoids the ambiguity of using 'lite' in two different senses? I'm happy to have another go at writing the original comment though if we'd prefer to avoid any actual refactors/renaming here -- lmk?

Reply:
I think that works! Its always going to be somewhat hard to understand I think but this is a good way of putting it.

Reply:
thanks! I'll merge now.